### DISCUSSION TITLE
Full time work

### DISCUSSION THREAD

**Discussion by shashkovdanil** [kodadot/nft-gallery#8418] [General]
Hi, Kodadot team!

During this month I worked a lot on Kodadot doing both big tasks (landing optimization, nuxt image, working on ssr) and small tasks (fixing small bugs). I respond quickly to all comments on PRs and generally deliver quality code. I already have 24 merged PRs and not a single closed one.

I have been proactive and offered improvements such as migration of UI kit to tailwindcss. I've participated in discussions in issues and a bit in code review.

I like the project, I like the processes here, I respect you very much for not being afraid to do open source development, very few products do open source development.

I would like to become a full part of your team if possible

cc @roiLeo @preschian @yangwao @vikiival

**Comment by shashkovdanil** [comment]
You can reach me by telegram shashkov0 or email at shashkovdanil@gmail.com

**Comment by yangwao** [comment]
Yes, I've noticed your work; gonna dm you :)
- https://github.com/kodadot/nft-gallery/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-desc+author%3Ashashkovdanil
